fabric/colors.py

```diff
@@ -22,13 +22,23 @@
 If ``bold`` is set to ``True``, the ANSI flag for bolding will be flipped on
 for that particular invocation, which usually shows up as a bold or brighter
 version of the original color on most terminals.
+
+It is now possible to disable color printing completely by setting the environment
+variable FABRIC_DISABLE_COLORS.  The value doesn't matter as long as it is
+Pythonically True.
+
 """
 
+import os
 
 def _wrap_with(code):
 
     def inner(text, bold=False):
         c = code
+
+        if os.environ.get('FABRIC_DISABLE_COLORS') is None:
```

Comment:
Jeez, yea. Thanks! I didn't even test this by hand :( Just did now to confirm. Pushed fix.